### PR TITLE
Support Rails 6.0

### PR DESCRIPTION
**Pull request by 9toon**
By the changes at https://github.com/rails/rails/pull/34468 , an issued SQL is moved into a separated property ( `.sql` ) on `StatementInvalid` from its body. So, we must check and transform `sql` as well as `message` .

please review 🙏 👀

### CODE REVIEW

**Comment by 9toon** [comment]
cc/ @cookpad/infra

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,18 @@ gemfile:
...
+  - gemfiles/rails_6.0.gemfile
 rvm:
   - 2.3.8
   - 2.4.6
   - 2.5.5
   - 2.6.3
+matrix:
+  exclude:
+    - rvm: 2.3.8
+      gemfiles/rails_6.0.gemfile
```
Comment (eagletmt) [inline .travis.yml]: syntax error

Comment (9toon) [reply .travis.yml]: oops 🙏 fixed and pushed -f 07cf7ff

File: lib/blouson/sensitive_query_filter.rb [hunk 2]
```diff
@@ -15,9 +15,10 @@ def self.filter_sensitive_words(message)
     end
 
     module StatementInvalidErrorFilter
-      def initialize(message = nil, original_exception = nil)
-        if SensitiveQueryFilter.contain_sensitive_query?(message)
+      def initialize(message = nil, original_exception = nil, sql: nil, binds: nil)
+        if SensitiveQueryFilter.contain_sensitive_query?(message) || (sql && SensitiveQueryFilter.contain_sensitive_query?(sql))
           message = SensitiveQueryFilter.filter_sensitive_words(message)
```
Comment (uasi) [inline lib/blouson/sensitive_query_filter.rb]: @9toon `message` が nil だとここでエラーが出ます。 `message = SensitiveQueryFilter.filter_sensitive_words(message) if message` にすればよさそうです。

Comment (9toon) [reply lib/blouson/sensitive_query_filter.rb]: fixed 👍  28334a2

**Comment by 9toon** [comment]
Could someone merge this PR(and release it)? because I don't have merge permission... 🙏

**Pull request merged**